lib/feedjira/core_ext/time.rb

```diff
@@ -18,7 +18,8 @@ def self.parse_safely(dt)
     elsif dt.respond_to? :to_s
       parse_string_safely dt.to_s
     end
-  rescue StandardError
+  rescue StandardError => e
```

Comment:
It's a little unclear, which exceptions could be thrown here, so for the moment I'll leave it on StandardError